package-lock.json

```diff
@@ -10024,7 +10024,7 @@
     },
     "semver-compare": {
       "version": "1.0.0",
-      "resolved": "https://packages.atlassian.com/api/npm/npm-remote/semver-compare/-/semver-compare-1.0.0.tgz",
```

Comment:
I don't know how did it get to the main branch. Seems repository override in .npmrc doesn't work on somebody's local copy

Reply:
yeah, that's kinda concerning...